text/0000-chmod.md

```diff
@@ -0,0 +1,62 @@
+# Chmod Buildpack
```

Comment:
First thing, I think this should be a Utilities buildpack, so the RFC can be a Utilities RFC. All that means, is just move this RFC proposal under the `text/utilities/` subdirectory.

Reply:
done